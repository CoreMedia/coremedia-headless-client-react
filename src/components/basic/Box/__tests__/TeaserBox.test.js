// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { TeaserBox } from '..';

describe('TeaserBox Component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<TeaserBox>Content</TeaserBox>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
