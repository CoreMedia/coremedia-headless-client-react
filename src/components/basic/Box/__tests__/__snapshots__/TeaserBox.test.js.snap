// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeaserBox Component should render correctly 1`] = `
.c0 {
  position: relative;
  border: 0;
  box-sizing: border-box;
  width: 100%;
  height: 100%;
}

.c0 > * {
  box-sizing: border-box;
}

<div
  className="c0"
>
  Content
</div>
`;
